lib/ts/index.ts

```diff
@@ -98,7 +99,8 @@ async function run() {
             --manager: Which package manager to use
             --autostart: Whether the CLI should start the project after setting up
         */
-        const userArgumentsRaw = (await yargs(hideBin(process.argv)).argv) as UserFlagsRaw;
+        let userArgumentsRaw = (await yargs(hideBin(process.argv)).argv) as UserFlagsRaw;
+        userArgumentsRaw = modifyUserArgumentsForAliasFlags(userArgumentsRaw);
```

Comment:
can we make this step cleaner? I don't like adding this sort of extra step in the middle. Hard to maintain.

Reply:
```jsx
export function modifyUserArgumentsForAliasFlags(userArguments: UserFlagsRaw): UserFlagsRaw {
    let _userArguments = structuredClone(userArguments);

    if (_userArguments.ui === UIBuildType.CUSTOM && _userArguments.frontend === "react") {
        _userArguments.frontend = "react-custom";
    }

    return _userArguments;
}
```
This is the above code which is being called, it is meant to replace the flags with their aliases. In our case, `react` is an alias for `react-custom` when the chosen UI Build Type is `custom`, this allows that the user can pass the flag `--frontend=react` for custom UI also, which improves the experience. Let me know if you think otherwise, we can discuss this. 

The reason was, for the custom UI the valid frontend option is `react-custom`, but we cannot assume that the end user will pass the required flag `--frontend=react-custom`, hence had to create a reusable function that replaces the alias values with the actual required values.